Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -714,11 +714,10 @@ public String newTask() {
         } catch (DataException e) {
             logger.error(e);
         }
-        save();
+        //save();
```

Comment:
Do not comment out obsolete code - remove it. You should squash this and your later delete commit to one.